app/AppKernel.php~

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
@PhotonSapiens 
this looks like a temporary file created by your editor - please remove it and don't commit anymore